splunk/resource_splunk_saved_searches_test.go

```diff
@@ -207,6 +207,7 @@ resource "splunk_saved_searches" "test" {
 	action_jira_service_desk_param_jira_summary = "error message"
 	action_jira_service_desk_param_jira_priority = "Normal"
 	action_jira_service_desk_param_jira_description = "test ticket creation"
+	action_jira_service_desk_param_jira_customfields = "customfield_10058":{"value":"custom_field_value_1"},"customfield_10046":{"value":"custom_field_value_2"}
```

Comment:
I believe this line may be why the test is failing here. Missing newline after the argument. We need the test 'TestAccSplunkSavedSearches' to pass before merging.
https://github.com/splunk/terraform-provider-splunk/actions/runs/10778298148/job/29994818441

Reply:
I suspect unescaped quotes are the reason for the failure. I believe the latest commit will fix. 🤞